librz/main/rz-find.c

```diff
@@ -633,6 +633,15 @@ RZ_API int rz_main_rz_find(int argc, const char **argv) {
 			return 1;
 		}
 		rzfind_open(&ro, file);
+
+		if (ro.exec_command) {
+			char *command = rz_str_newf("%s %s", ro.exec_command, file);
+			int status = system(command);
```

Comment:
should we use execv* ? anyway use `rz_sys_system` and not `system`

Reply:
I thought of that but went with using `system`, will make the changes soon.